react/ComponentsGrid.tsx

```diff
@@ -1,16 +1,14 @@
-import React, { Fragment, FunctionComponent } from 'react'
+import React, { FunctionComponent } from 'react'
 import { FormattedMessage, defineMessages } from 'react-intl'
-import { Helmet, NoSSR, withRuntimeContext } from 'vtex.render-runtime'
+import { withRuntimeContext } from 'vtex.render-runtime'
```

Comment:
same

Reply:
I also need this one for the same reason hahaha GitHub should show the whole file

Reply:
Removed :)